app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
+    pass
+
+
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
```

Comment:
NotVaccinatedError and OutdatedVaccineError should have __init__() to use inherited __init__() from a base class:

Example:
class OutdatedVaccineError(VaccineError):
    def __init__(self, message):
        super.__init__(message)